tests/unit/add-one-test.js

```diff
@@ -0,0 +1,25 @@
+var simple = require('simple-mock')
+var test = require('tape')
+
+var addOne = require('../../lib/helpers/add-one')
+
+test('add-one non-409 error', function (t) {
+  t.plan(1)
+
+  var non409Error = new Error('not a 409 error')
+  var state = {
+    db: {
+      put: simple.stub().rejectWith(non409Error)
+    }
+  }
+
+  addOne(state, {})
```

Comment:
I guess this is technically an integration test, since we're calling `addTimestamp`.  I wanted to mock out `addTimestamp` but I couldn't figure out how to do it.  I think we used to use `proxyquire` for that?  Is there a way to do that with `simple-mock`?